### PR TITLE
Trimming extensions can be disabled in settings.

### PR DESCRIPTION
**Pull request by uncletrunks**
Signed-off-by: uncletrunks <uncletrunks@protonmail.com>

### CODE REVIEW

File: src/gmenu2x.cpp [hunk 1]
```diff
@@ -668,6 +669,10 @@ void GMenu2X::showSettings() {
 			*this, tr["Output logs"],
 			tr["Logs the output of the links. Use the Log Viewer to read them."],
 			&confInt["outputLogs"])));
+	sd.addSetting(unique_ptr<MenuSetting>(new MenuSettingBool(
+			*this, tr["Trim extensions"],
+			tr["Trims extensions in the file lister."],
```
Comment (pcercuei) [inline src/gmenu2x.cpp]: `Trims extensions in the file lister.` -> `Trim file extensions in the file explorer.`

**Comment by uncletrunks** [comment]
Amended.

**Comment by pcercuei** [comment]
`file lister` -> `file explorer` please :)

**Comment by pcercuei** [comment]
I'm annoying, but... `Trim file extensions` -> `Trim extensions` :)
Then I think I can merge.

**Comment by uncletrunks** [comment]
> I'm annoying, but... `Trim file extensions` -> `Trim extensions` :)
> Then I think I can merge.

That was the specific wording in your original revision but I gotcha, should be good now.

**Comment by pcercuei** [comment]
I know, I just changed my mind :) 

Thanks for the patch!

**Pull request merged**